assets/js/components/ChecksCatalog/ChecksCatalog.jsx

```diff
@@ -26,10 +24,13 @@ function ChecksCatalog({ catalogData, catalogError, loading, updateCatalog }) {
     <>
       <div className="flex">
         <PageHeader className="font-bold">Checks catalog</PageHeader>
-        <ProviderSelection
+        <Select
+          optionsName="providers"
           className="ml-auto"
-          providers={Object.keys(updatedProvider)}
-          selected={selectedProvider}
+          options={PROVIDERS}
+          withAllOption
```

Comment:
```suggestion
          options={[ALL_OPTION, ...PROVIDERS]}
```

Reply:
> we could reuse this component as a generic select in the future and not only as a filter.
We can already do this, even with the `withAllOption` prop, right?